path_login.go

```diff
@@ -176,7 +177,9 @@ func (b *jwtAuthBackend) pathLogin(ctx context.Context, req *logical.Request, d
 			Time:    time.Now(),
 		}
 
-		if err := claims.Validate(expected); err != nil {
+		// We bake leeway into expiration/not before, so we pass in a duration of 0
```

Comment:
Good catch about the double-leeway, but that will only apply if our calculations are used. If the JWT has a `iat`/`nbf` and `exp`, we'll now be checking it with no leeway, when before the standard library amount was used. We should probably have a default value set to `jwt.DefaultLeeway`, and then reset that to 0 if we end up doing out our exp/nbf calcs.